src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Notifiers\Slack;
+
+use App\Entity\Conference;
+use App\Entity\Submit;
+use App\Entity\Talk;
+use App\Repository\ConferenceRepository;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SlackNotifier
+{
+    private string $webHookUrl;
+    private string $env;
+    private HttpClientInterface $httpClient;
+    private RouterInterface $router;
+    private ConferenceRepository $conferenceRepository;
+    private SlackBlocksBuilder $slackBlocksBuilder;
+
+    public function __construct(string $webHookUrl, string $env, HttpClientInterface $httpClient, RouterInterface $router, ConferenceRepository $conferenceRepository, SlackBlocksBuilder $slackBlocksBuilder)
+    {
+        $this->webHookUrl = $webHookUrl;
+        $this->env = $env;
+        $this->httpClient = $httpClient;
+        $this->router = $router;
+        $this->conferenceRepository = $conferenceRepository;
+        $this->slackBlocksBuilder = $slackBlocksBuilder;
+    }
+
+    /** @param array<Conference> $newConferences */
+    public function sendDailyNotification(array $newConferences): void
+    {
+        $conferences = $this->buildConferencesBlocks($newConferences);
+        $endingCfps = $this->buildCfpsBlocks();
+
+        $blocks = [
+            'blocks' => [
+                ...$conferences,
+                ...$endingCfps,
+                $this->slackBlocksBuilder->buildDivider(),
+            ],
+        ];
+
+        $this->notify($blocks);
+    }
+
+    /** @param array<Submit> $submits */
+    public function sendNewTalkSubmittedNotification(array $submits, Talk $talk): void
+    {
+        $header = $this->slackBlocksBuilder->buildHeader('🗣  New submitted talk');
+        $title = $this->slackBlocksBuilder->buildSimpleSection(sprintf('*Title :* %s', $talk->getTitle()));
+        $intro = $this->slackBlocksBuilder->buildSimpleSection(sprintf('*Introduction :* %s', $talk->getIntro()));
+
+        $blocks = [
+            'blocks' => [
+                $header,
+                $this->slackBlocksBuilder->buildDivider(),
+                $title,
+                $intro,
+            ],
+        ];
+
+        if (0 < \count($submits)) {
+            $submittedText = '*Submitted for :*';
+            $index = 1;
+
+            foreach ($submits as $submit) {
```

Comment:
```suggestion
            foreach ($submits as $k => $submit) {
```
Here, you can use $k as index instead of create a `$index` variable and increment it (do a +1 on $k if needed)